### PR TITLE
Desktop-version

### PR DESCRIPTION
**Pull request by KAYEMBAJONATHAN**
.This is my desktop version 
.Update README. md
.I Readapt the style.css
.fix html file add some new img

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -88,7 +89,7 @@ <h1>I'm Jonathan Glad to see you</h1>
 
   <section class="project-container">
     <div class="project-card">
-      <img src="./assets/Snapshoot Portfolio-2.png" alt="project card image" />
+      <img src="./assets/Snapshoot Portfolio.png" alt="project card image" />
```
Comment (wayungi) [inline index.html:92]: - [ ] For all your project cards, they are not fully matching the figma design. Please take a look at the images below to compare and contrast and the fix the issues by tweaking your margin or padding to match the design.

**figma**
des1
**Your implementation**
des2

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @KAYEMBAJONATHAN , 
image

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Between your projects and the `about myself` section, your background color cannot be seen. Please add it back. check image below for clarification.
**design**
dez1
**Your implementation**
dez2

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -88,7 +89,7 @@ <h1>I'm Jonathan Glad to see you</h1>
 
   <section class="project-container">
```
Comment (Zuhaib042) [inline index.html:90]: - [ ] As the previous reviewer mentioned about the background-color of the sections and cards should be similar to the figma design. The origin of the problem for this is line no `47` in your style.css where you are styling your `section tag` generally. kindly consider removing that line and then give the `hero section` and the `about me` section background-color of `#fff`
your design
re
fd

figma design
qw
zx

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @KAYEMBAJONATHAN , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
close

 ## Highlights
✅ Professional Readme
✅ Professional PR
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @KAYEMBAJONATHAN , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] No linter errors :white_check_mark: 
- [x]  GitHub flow is used correctly :white_check_mark: 
- [x] Implemented all the changes from the previous reviewer :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**